python/tidy/servo_tidy/tidy.py

```diff
@@ -991,25 +1001,27 @@ def check_commits(path='.'):
     raise StopIteration
 
 
-def scan(only_changed_files=False, progress=True):
+def scan(only_changed_files=False, progress=True, stylo=False):
```

Comment:
I'm saying that we should override it inside the `scan` function when we have `stylo`. `FileList` iterator should bail out when it doesn't find git, regardless of whether we're in stylo or not.